netlify/utils/stelace.js

```diff
@@ -0,0 +1,20 @@
+// Copied from scripts/sdk.js using ES6 import/export syntax
+
+import { createInstance } from 'stelace'
+import URL from 'url-parse'
+
+export function initStelaceSdk ({ apiBaseURL, apiKey }) {
+  const stelace = createInstance({ apiKey })
+
+  if (apiBaseURL) {
+    const parsedUrl = new URL(apiBaseURL)
+
+    const host = parsedUrl.hostname
+    const port = parsedUrl.port
+    const protocol = parsedUrl.protocol.slice(0, -1)
+
+    stelace.setHost(host, port, protocol)
+  }
+
+  return stelace
+}
```

Comment:
This is the fourth file of this kind. Do you think you can refactor using `initStelaceSdk` from `src/util/stelace.js` for DRYer code?

Reply:
My bad. I missed the file `src/util/stelace.js` that is in ES6 module.